packages/styles/settings-tools/_s.badges.scss

```diff
@@ -0,0 +1,28 @@
+$badges: get-token(color, badge);
+
+$badge-border: get-border('s');
+
+@mixin set-badge-base {
+  @include set-border-radius();
+
+  border: $badge-border solid transparent;
+  display: inline-flex;
+  align-items: center;
+  padding: 0 $mu050;
+  margin: 0;
+  vertical-align: middle;
+  min-height: 1.375rem;
+  flex-shrink: 0;
+}
+
+@mixin set-badge-theme($props) {
+  color: map-get($props, 'text');
+
+  @if map-has-key($props, 'border') {
+    border-color: map-get($props, 'border');
+  }
+
+  @if map-has-key($props, 'background') {
+    background-color: map-get($props, 'background');
+  }
```

Comment:
~~@if map-has-key($props, 'border')~~
~~@if map-has-key($props, 'background')~~

As of there are no borderless versions of badges, it is not necessary to test for the presence of the value.
So you can directly write your code as follow:

```scss
@mixin set-badge-theme($props) {
  color: map-get($props, "text");
  border-color: map-get($props, "border");
  background-color: map-get($props, "background");
}
```

Reply:
https://github.com/adeo/mozaic-design-system/pull/919/commits/b11ce0cc6adbe2ad4318a29edc90dfc739b7e77c